Makefile

```diff
@@ -59,10 +59,12 @@ lint: test-sec
 	cp $(crd_file) $(crd_tmp); make crd; if ! diff -q $(crd_file) $(crd_tmp); then mv $(crd_tmp) $(crd_file); exit 1; else rm $(crd_tmp); fi
 
 .PHONY: test-sec
-test-sec:
-	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
+test-sec: $GOPATH/bin/gosec
 	gosec -severity medium --confidence medium -quiet ./...
 
+$GOPATH/bin/gosec:
```

Comment:
I wonder if this is worth doing upstream as well, since it will help developers running the tests locally. It also means we won't have to install it for every travis target, only the ones that use it (not such a big deal for gosec, but doing the same for the operator-sdk would save a lot of time). Plus, fewer merge conflicts in openshift.